Project/Assets/Prairie/Framework/Script/Twine/TwineNode.cs

```diff
@@ -15,14 +19,43 @@ public class TwineNode : MonoBehaviour {
 	[HideInInspector]
 	public string[] childrenNames;
 	public List<GameObject> parents = new List<GameObject> ();
+	public bool isDecisionNode;
 
 	private bool isMinimized = false;
+	private bool isOptionsGuiOpen = false;
+
+	private int selectedOptionIndex = 0;
+
+	void Start () 
+	{
+		// Upon creation of this node, ensure that it is a decision node if it has
+		//	the decision tag:
+		this.isDecisionNode = this.tags.Contains (PRAIRIE_DECISION_TAG);
+	}
 
 	void Update ()
 	{
 		if (Input.GetKeyDown (KeyCode.Tab))
 		{
-			this.ToggleMinimize ();
+			this.isMinimized = !this.isMinimized;
+		}
+
+		if (this.isDecisionNode) {
+			if (this.isOptionsGuiOpen && Input.GetKeyDown (KeyCode.Q)) {
+				// Press Q to scroll through the children nodes
+				this.selectedOptionIndex = (this.selectedOptionIndex + 1) % (children.Length);
+			} else if (this.isOptionsGuiOpen && (Input.GetKeyDown (KeyCode.KeypadEnter) || Input.GetKeyDown (KeyCode.Return))) {
+				// Press ENTER or RETURN to select that child
+				this.ActivateChildAtIndex (selectedOptionIndex);
```

Comment:
I think we should put a check in and to ensure this is the currently activated node, before we risk modifying any state based on user input. I don't know - this just worries me since update gets run on every twine node (not just the active one).

I'm pretty sure that any node with `.enabled = false` won't run their `Update()` method (so this isn't really a problem), but a sanity check would go a long way in making it clearer.

If we later decide that twine node's should have a `isCurrentNode` property (instead of piggybacking off of `.enabled`, which has side effects), a bug would be introduced here.